main.go

```diff
@@ -76,7 +74,7 @@ func scytale(arguments []string) int {
 		f = pflag.NewFlagSet(applicationName, pflag.ContinueOnError)
 		v = viper.New()
 
-		logger, metricsRegistry, webPA, err = server.Initialize(applicationName, arguments, f, v, webhook.Metrics, aws.Metrics, basculechecks.Metrics, basculemetrics.Metrics, chrysom.Metrics, Metrics, service.Metrics)
+		logger, metricsRegistry, webPA, err = server.Initialize(applicationName, arguments, f, v, webhook.Metrics, aws.Metrics, basculechecks.Metrics, basculemetrics.Metrics, consul.Metrics, Metrics, service.Metrics)
```

Comment:
did we mean to add consul metrics in here? just double-checking as it wasn't there before

Reply:
Yup! Previously, the chrysom package declared metrics in the old webpa-common/xmetrics style.  Now that chrysom doesn't supply old style metrics, consul has to declare the metrics itself for the old way of registering metrics.

Reply:
Just saw that, great!